playwright.config.ts

```diff
@@ -90,8 +90,11 @@ export default defineConfig({
     // },
   ],
   webServer: {
-    command: process.env.CI ? '' : 'mvn gwt:run',
+    command: process.env.CI
+      ? 'echo SWC not available at baseURL'
+      : 'mvn gwt:run',
```

Comment:
Before running e2e tests, Playwright makes a request to the baseURL to determine if the server is ready. If the server is not ready, Playwright attempts to run the `command` to start a new server. Passing an empty string here resulted in an obscure error (`TypeError: The argument 'file' cannot be empty. Received ''`); instead, output a more useful message for troubleshooting in CI.